### PR TITLE
Change behavior of navigation Previous/Next buttons

### PR DESCRIPTION
**Pull request by idegtiarov**

### CODE REVIEW

File: static/js/textbox.html [hunk 1]
```diff
@@ -12,6 +12,36 @@
             return vars;
         }
 
+        function disableNavButtons(){
+            var next_buttons = window.parent.document.getElementsByClassName('button-next');
+            var previous_buttons = window.parent.document.getElementsByClassName('button-previous');
+            var i;
+            for (i = 0; i < next_buttons.length; i ++){
```
Comment (z4y4ts) [inline static/js/textbox.html]: Use `nextButtons.forEach` here

File: static/js/textbox.html [hunk 2]
```diff
@@ -12,6 +12,36 @@
             return vars;
         }
 
+        function disableNavButtons(){
+            var next_buttons = window.parent.document.getElementsByClassName('button-next');
```
Comment (z4y4ts) [inline static/js/textbox.html]: Use camelCase

File: static/js/textbox.html [hunk 3]
```diff
@@ -12,6 +12,36 @@
...
+            var previous_buttons = window.parent.document.getElementsByClassName('button-previous');
```
Comment (z4y4ts) [inline static/js/textbox.html]: Use camelCase

File: static/js/textbox.html [hunk 4]
```diff
@@ -12,6 +12,36 @@
...
+                next_buttons[i].setAttribute('disabled', 'disabled');
+                previous_buttons[i].classList.add('disabled');
+                previous_buttons[i].setAttribute('disabled', 'disabled');
+            }
+        }
+
+        function enableNextButtons(problemLink) {
+            var next_buttons = window.parent.document.getElementsByClassName('sequence-nav-button button-next');
+            var i;
+            console.log(problemLink);
```
Comment (z4y4ts) [inline static/js/textbox.html]: Delete

File: static/js/textbox.html [hunk 5]
```diff
@@ -12,6 +12,36 @@
...
+            for (i = 0; i < next_buttons.length; i ++){
+                next_buttons[i].classList.remove('disabled');
+                next_buttons[i].removeAttribute('disabled', 'disabled');
+                var clone = next_buttons[i].cloneNode();
```
Comment (z4y4ts) [inline static/js/textbox.html]: Why do we need this `clone`?

File: static/js/textbox.html [hunk 6]
```diff
@@ -12,6 +12,36 @@
...
+                previous_buttons[i].classList.add('disabled');
+                previous_buttons[i].setAttribute('disabled', 'disabled');
+            }
+        }
+
+        function enableNextButtons(problemLink) {
+            var next_buttons = window.parent.document.getElementsByClassName('sequence-nav-button button-next');
+            var i;
+            console.log(problemLink);
+            for (i = 0; i < next_buttons.length; i ++){
```
Comment (z4y4ts) [inline static/js/textbox.html]: Same thing with camelCase and `forEach`.

File: static/js/textbox.html (hunk 2 above)
Comment (idegtiarov) [inline static/js/textbox.html]: done

File: static/js/textbox.html (hunk 3 above)
Comment (idegtiarov) [inline static/js/textbox.html]: done

File: static/js/textbox.html (hunk 1 above)
Comment (idegtiarov) [inline static/js/textbox.html]: Here we manipulate with nextButtons array and previousButtons array so forEach method isn't suitable.

File: static/js/textbox.html (hunk 4 above)
Comment (idegtiarov) [inline static/js/textbox.html]: done

File: static/js/textbox.html (hunk 6 above)
Comment (idegtiarov) [inline static/js/textbox.html]: good catch, thanks. Done

File: static/js/textbox.html (hunk 5 above)
Comment (idegtiarov) [inline static/js/textbox.html]: there is no other way to remove or disable Event listeners registered earlier than our one

Comment (z4y4ts) [inline static/js/textbox.html]: Ok, I see. Please add a comment about this.

File: static/js/textbox.html [hunk 7]
```diff
@@ -12,6 +12,35 @@
...
+                nextButtons[i].classList.add('disabled');
+                nextButtons[i].setAttribute('disabled', 'disabled');
+                previousButtons[i].classList.add('disabled');
+                previousButtons[i].setAttribute('disabled', 'disabled');
+            }
+        }
+
+        function enableNextButtons(problemLink) {
+            var nextButtons = window.parent.document.getElementsByClassName('sequence-nav-button button-next');
+            var i;
```
Comment (z4y4ts) [inline static/js/textbox.html]: Unused variable

**Pull request merged**